Formula/hive.rb

```diff
@@ -29,6 +29,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert_match /default/, shell_output("#{bin}/hive -e 'show databases;'")
+    system "#{bin}/schematool -initSchema -dbType derby"
+    assert_match /Hive 2.0.0/, shell_output("#{bin}/hive --version")
```

Comment:
Please don't use a regular expression match where there's no need for one. Currently, this would also match, e.g., `Hive 230.0`. Better yet, to make this work for future updates, too, please make this:

```ruby
assert_match "Hive #{version}", shell_output("#{bin}/hive --version")
```